app/src/editor/canvas/components/Ports/Value/Select/index.jsx

```diff
@@ -49,7 +49,7 @@ const Select = ({
                 {/* `select` holding a currently selected value. This hidden (`visibility: hidden`) control
                     dictates the width of the actual (visible) control above. */}
                 <select className={styles.spaceholder}>
-                    <option>{optionMap.get(value)}</option>
+                    <option>{optionMap.get(value) || value}</option>
```

Comment:
should probably handle case where `optionMap.get(value)` is `0`

Reply:
Done.